test/Api.IntegrationTest/SecretsManager/Controllers/AccessPoliciesControllerTest.cs

```diff
@@ -91,9 +153,100 @@ public async Task CreateProjectAccessPolicies()
     }
 
     [Fact]
-    public async Task UpdateAccessPolicy()
+    public async Task CreateProjectAccessPolicies_NoPermission()
     {
-        var initData = await SetupAccessPolicyRequest();
+        // Create a new account as a user
+        var (org, _) = await _organizationHelper.Initialize(true, true);
+        var (email, orgUser) = await _organizationHelper.CreateNewUser(OrganizationUserType.User, true);
+        await LoginAsync(email);
+
+        var initialProject = await _projectRepository.CreateAsync(new Project
```

Comment:
```suggestion
        var project = await _projectRepository.CreateAsync(new Project
```

Reply:
Resolved in https://github.com/bitwarden/server/pull/2628/commits/79bce819b50d75b5ce021b4b4dfad21dc9c97cd8